distinguish between interfaces complying to Article 5a common protoco… (#1)
paulbastian/eudi-doc-architecture-and-reference-framework: docs/arf.md

    (QES) Remote Service Provider. This interface is specifically used
    to execute a QES remote signature.

> Furthermore, the following interfaces are subject to harmonization, but are

Reviewer: I'm not sure I understand the difference between harmonization and standardization. I would say those interfaces could be listed to establish a certain terminology. The design is at the discretion of the wallet provider.

> Furthermore, the following interfaces are subject to harmonization, but are
> not considered as interoperable, standardized protocols:

Author: ```suggestion
Furthermore, the following interfaces are designed by the Wallet Provider, but
are not considered as interoperable, standardized protocols: 
```

-   **The Wallet Provider Interface (WPI)** is used by the Wallet
    Instance to communicate with the Wallet Provider to issue the Wallet
    Trust Evidence and the Wallet Instance Attestation, as well as
    provide support to the user and collect metrics for accounting
    purposes.

> -   **The Secure Cryptographic Interface (SCI)** enables the Wallet

Reviewer: I think a definition, at least, of the capabilities of this interface makes sense. It would allow to build wallet apps on top of different kinds of WSCDs.
